k8s/deployments/forum_deployment.yaml

```diff
@@ -0,0 +1 @@
+apiVersion: apps/v1
kind: Deployment
metadata:
  name: forum-app
spec:
  replicas: 1
  selector:
    matchLabels:
      app: forum-app
  template:
    metadata:
      labels:
        app: forum-app
    spec:
      containers:
      - name: forum-app
        image: saberan/forum_app:latest
        ports:
        - containerPort: 8000
        command: ["python", "manage.py", "runserver", "0.0.0.0:8000"]
        envFrom:
        - configMapRef:
            name: forum-config
        env:
        - name: DB_NAME
          valueFrom:
            secretKeyRef:
              name: forum-secrets
              key: DB_NAME
        - name: DB_USER
          valueFrom:
            secretKeyRef:
              name: forum-secrets
              key: DB_USER
        - name: DB_PASSWORD
          valueFrom:
            secretKeyRef:
              name: forum-secrets
              key: DB_PASSWORD
        - name: DB_HOST
          value: "db"
        - name: DB_PORT
          value: "5432"
        - name: DJANGO_SECRET_KEY
          valueFrom:
            secretKeyRef:
              name: forum-secrets
              key: DJANGO_SECRET_KEY
        - name: ADMIN_EMAIL
          valueFrom:
            secretKeyRef:
              name: forum-secrets
              key: ADMIN_EMAIL
        - name: ADMIN_PASSWORD
          valueFrom:
            secretKeyRef:
              name: forum-secrets
              key: ADMIN_PASSWORD

```

Comment:
![Screenshot 2024-11-08 222538](https://github.com/user-attachments/assets/77922aac-d379-4ad3-b40a-b9493d9ae5d4)
I see a problem with this file

Reply:
fixed
done